### ISSUE TITLE
Feature Request:  Add support for EfficientAD from Anomalib

### ISSUE THREAD

**Issue by j99ca** [open-edge-platform/training_extensions#3971]
Currently there are only 2 supported anomaly models in OTX:  PADIM and STFPM.

EfficientAD is a newer model that that Anomalib supports that has very high accuracy according to papers with code as well as my own internal testing for industrial use-cases. This would be a great model to include in OTX's anomaly detection flow, or perhaps tighter integration between OTX and Anomalib in general would be ideal.

**Comment by harimkang** [comment]
I personally would like to see all of Anomalib's models evolve to be commonly used in OTX, just like Timm or hugging-face models are supported by wrapper classes. But as far as I know, this needs some cleanup at the moment. But I think it's achievable.
@ashwinvaidya17

**Comment by ashwinvaidya17** [comment]
Yeah. That's the plan. We are going to slowly all support for all the algorithms available in Anomalib
